Sources/MongoSwift/MongoCollection+FindAndModify.swift

```diff
@@ -206,6 +210,8 @@ public struct FindOneAndDeleteOptions: FindAndModifyOptionsConvertible, Decodabl
         try FindAndModifyOptions(
             collation: self.collation,
             hint: self.hint,
+            // swiftlint:disable colon
```

Comment:
what is this disabling?

Reply:
Theres a lint rule that wants there to be nothing in between the `let` and the `:`